.github/workflows/weekly-pulumi-update.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         goversion: [1.17.x]
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Is there any major changes here we need to test first before upgrading to v3?

Reply:
Not that I can see, but would appreciate your thoughts: https://github.com/actions/checkout/releases/tag/v3.0.0. I bumped it so we could use the latest of https://github.com/peter-evans/create-pull-request.